src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -73,4 +119,322 @@ public void userSeesDownloadFiles(final String expectedLength) {
         Select listBox = new Select(selectElement);
         assertEquals(listBox.getOptions().size(), Integer.parseInt(expectedLength));
     }
+    /**
+     * Select 2015 edition products file from drop down and download .xml file.
+     * @throws Exception - to handle IOException with FileUtil
+     */
+
+    @When("^I download the 2015 edition products file$")
+    public void download2015editionProductsFile() throws Exception {
+        /**
+         * Clean all files from download directory so each time there is only latest
+         * download file to read.
+         */
+        this.movefiles();
+        FileUtils.cleanDirectory(dir);
+        ChplDownloadPage.downloadoption2015editionProductsFile(driver).click();
+        ChplDownloadPage.downloadFileButton(driver).click();
+    }
+    /**
+     * Assert filename of download file.
+     */
+    @Then("^the downloaded file shows chpl-2015-currentdate.xml filename$")
+    public void verify2015editionProductsFilename() {
+        /**
+         * Get latest download file in the directory. Keep checking until download is
+         * complete then read final filename.
+         */
+
+        String dwldFileName = "";
+        int loopBack = 0;
```

Comment:
Using an `int` here is weird. It would make more sense (and be more readable) if it was a boolean. Something like `boolean fileFound = false;` then in the "if" block below you can just do `if finalName.equals: fileFound = true`, and you won't need the else at all.

Similarly with the other instances of this pattern below.